App/Lettuce/Sources/Lettuce/MessageCenter/Requests/messaging/message-attachments/DownloadAttchmentRequest.swift

```diff
@@ -0,0 +1,25 @@
+import Foundation
+
+import HTTPTypes
+
+import Onion
+
+//https://developer.allegro.pl/documentation#operation/uploadAttachmentPUT
+public struct DownloadAttchmentRequest {
+
+    let attachmentId: Identifier<Attachment>
+
+    var path: String { "/messaging/message-attachments/\(attachmentId.id)" }
+
+    var method: HTTPRequest.Method { .get }
+
+    var headerFields: HTTPFields {
+        var fields: HTTPFields = [:]
+
+        fields[token.httpField.name] = token.httpField.value
```

Comment:
hint: I think that this can be a bit shorter:

```swift
var headerFields: HTTPFields { 
    [token.httpField.name: token.httpField.value] 
}
```

Maybe we need some better API to do this :)